HTML_files/templating/app/views.py

```diff
@@ -24,7 +24,7 @@ def call_endpoint(method, endpoint,
 def get_auth_token(email, password):
     return (call_endpoint('authtoken', 'get', (email, password))['token'], '')
 
-BASE_URL = 'http://localhost:5000/'
+BASE_URL = 'http://localhost:5001/'
```

Comment:
What's with the hardcoded urls and ports here?

Reply:
it's a good question! the answer is probably "2016 burton didn't know how to make this nicer" — and 2023 burton didn't want to make any dramatic changes or improvements until builds are working, tests are in place, and all that. i'll add a TODO to see about fixing this :)